src/styles/_catalog.scss

```diff
@@ -1,22 +1,15 @@
 .catalog {
```

Comment:
You can call the file just `catalog.scss`. No need to underline if it's a file for block